packages/svelte/tests/validator/samples/runes-referenced-nonstate-bind-this/input.svelte

```diff
@@ -1,11 +1,23 @@
 <script>
```

Comment:
Here's a [REPL link](https://svelte-5-preview.vercel.app/#H4sIAAAAAAAAE42S3WrDMAyFX8V4vUggNCy9S5PAnmMZo7HlzVsqhVgJGybvPtz0J7Td2JXxx5F0DpKXxrbgZP7sJe72IHP51HUykfzdhY8boWWQiXQ09CqQwqnedlzVWHMLLJBeEUA_bq9AFsARaQJ3pTqj7BZtRClWjncMUbytsUgvI9EHY2QWLac7MLsHN1NoUGg7isaizvndutIvGlVFqu04j3mwRnA_QKjh65pT5kXFb5rsL83C739Um4vKp9bMcZqBmVAQqtaqz9JHsSgroQgdtbBu6S06OYmnyhAV6VxR1SgTuSdtjQUt80PY5HwFjgdj1uf1H8_hwy1PAb466lmYARVbQhGGqSYWPqRYgTGgOJr9HFDNS1uqieL4sPwpPNOtm5fpB1GfnjWfAgAA) so that you can see which of them would currently produce a warning.